### ISSUE TITLE
[4.0] com_finder - announce search results as soon as possible

### ISSUE THREAD

**Issue by hans2103** [joomla/joomla-cms#30536]
> When you enter a search term and press the search button the page reloads and anyone using a screen reader  will have to navigate all the way down the page and past the search form in order to find out if there are any results. They should be taken to the results immediately OR there should be an announcement that there are x results on page load. Its even worse when the advanced link (it should be a button) is selected

_Originally posted by @brianteeman in https://github.com/joomla/joomla-cms/pull/30528#issuecomment-684430237_

**Comment by hans2103** [comment]
I can relate on this. 
Would the a11y be improved when a `role=status` is added to PR https://github.com/joomla/joomla-cms/pull/30535

**Comment by brianteeman** [comment]
You cant use status as that is for live regions ie parts of the page that are updated
In our case the page is reloaded and not updated

role=alert should satisfy the requirement

**Comment by hans2103** [comment]
added to pr #30535 with commit https://github.com/joomla/joomla-cms/pull/30535/commits/28542706e67ac49c699826fe7381e3bf53e77865
